dispatch/glproc.py

```diff
@@ -573,6 +576,9 @@ def failFunction(self, function):
             print(r'    if (messageLog != 0 && bufsize > 0) *messageLog = 0;')
             print(r'    return 0;')
             return
+        if function.name.startswith('eglQueryDebugKHR'):
+            print(r'    if (value != 0) *value = 0;')
+            return
```

Comment:
`eglQueryDebugKHR` returns a `EGLBoolean`.  You're missing a

```python
    print(r' return EGLFalse;')
```

Ditto for all the other functions that return values .

This should fix the MSVC builds.

Reply:
Thanks for then update.  I mistyped, it should have been `return EGL_FALSE`.  I know 0 and `EGL_FALSE` is all the same from the compiler POV, but it's confusing from my POV.